src/libs/focusWithDelay.ts

```diff
@@ -1,10 +1,11 @@
 import {InteractionManager, TextInput} from 'react-native';
 import ComposerFocusManager from './ComposerFocusManager';
 
+type FocusWithDelayReturn = (shouldDelay?: boolean) => void;
```

Comment:
```suggestion
type FocusWithDelay = (shouldDelay?: boolean) => void;

```

To follow the patterns we have been using, just `FocusWithDelay` is enough 🙂

Reply:
@fabioh8010 fixed 😄